app/config/config_dev.yml

```diff
@@ -3,14 +3,15 @@
 
 imports:
     - { resource: "config.yml" }
+    - { resource: "parameters_dev.yml" }
```

Comment:
Hmm, it would be nice not to have it, but we will have to provide the correct hostname anyway :/